plugins/CorePluginsAdmin/angularjs/form-field/form-field.directive.js

```diff
@@ -281,6 +281,10 @@
                 function formatPrettyDefaultValue(defaultValue, availableOptions) {
 
                     if (!angular.isArray(availableOptions)) {
+                        if (angular.isArray(defaultValue)) {
```

Comment:
If I see that correctly, the defaultValue is always a string here, as it isn't converted at any point.
Guess it would need something like this when getting the defaultValue:

```php
if (field.uiControl === 'multituple' && angular.isString(defaultValue) && defaultValue) {
   defaultValue = JSON.parse(defaultValue);
}
```

Reply:
@sgiehl did that work for you? It still serialized it to json afterwards for me even though it did JSON.parse it (tested with debugger)

![image](https://user-images.githubusercontent.com/273120/42782023-c7b41c1c-899b-11e8-9bf1-1649c76b0e31.png)

Reply:
didn't test that before. but really doesn't work as expected. But can't figure out why right now...